var/spack/repos/builtin/packages/py-csscompressor/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCsscompressor(PythonPackage):
+    """A python port of YUI CSS Compressor."""
+
+    pypi = "csscompressor/csscompressor-0.9.5.tar.gz"
+
+    #maintainers = ["wscullin"]
+
+    version("0.9.5", sha256="afa22badbcf3120a4f392e4d22f9fff485c044a1feda4a950ecc5eba9dd31a05")
+    
+    depends_on("py-setuptools", type="build")
+
+    depends_on("py-argparse", when="^python@:2.6,3.0:3.1", type=("build", "run"))
```

Comment:
You can actually remove the argparse dep now, I'm removing the package entirely in #33898. Sorry for the confusion!